packages/eigen/eigen.0.3.2/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "Liang Wang <ryanrhymes@gmail.com>"
+authors: [ "Liang Wang" ]
+license: "MIT"
+homepage: "https://github.com/owlbarn/eigen"
+dev-repo: "git+https://github.com/owlbarn/eigen.git"
+bug-reports: "https://github.com/owlbarn/eigen/issues"
+doc: "https://owlbarn.github.io/eigen/eigen"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.02"}
```

Comment:
```suggestion
  "ocaml" {>= "4.04"}
```
The configurator script uses `String.split_on_char`. This is fine since owl requires 4.10 in any case